src/core/ext/filters/client_channel/lb_policy.h

```diff
@@ -149,7 +149,9 @@ void grpc_lb_policy_init(grpc_lb_policy *policy,
 
 /** Finds an appropriate subchannel for a call, based on \a pick_args.
 
-    \a target will be set to the selected subchannel, or NULL on failure.
+    \a target will be set to the selected subchannel, or NULL on failure
+    or when the LB policy decides to drop the call.
```

Comment:
I wonder if we'd want to have a different way to signal dropping the call. Using `NULL` for both cases makes it ambiguous.